src/Progress/MVVM/ViewModelBase.cs

```diff
@@ -20,154 +20,18 @@
 
 using System;
 using System.ComponentModel;
-using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.Runtime.CompilerServices;
-using System.Threading.Tasks;
-using System.Windows.Threading;
 
 namespace SonarLint.VisualStudio.Progress.MVVM
 {
     public class ViewModelBase : INotifyPropertyChanged
     {
-        private readonly Dispatcher dispatcher;
-
-#if DEBUG
-        private readonly string stackPrint;
-#endif
-        protected ViewModelBase()
-        {
-            this.dispatcher = Dispatcher.CurrentDispatcher;
-
-#if DEBUG
-            this.stackPrint = new StackTrace().ToString();
-#endif
-        }
-
         /// <summary>
         ///  Occurs when a property value changes.
         /// </summary>
         public event PropertyChangedEventHandler PropertyChanged;
 
-        /// <summary>
```

Comment:
I wasn't looking forward to cleaning this code up. Had it been difficult to clean up I would have left it alone since it is only in the `Progress` library, which is only used by our UI in the `Team Explorer`. When the binding UI is rewritten, the UI won't be hosted in the `Team Explorer`, and this project can then be dropped.